src/plugins/home/public/application/components/home_app.js

```diff
@@ -92,7 +93,7 @@ export function HomeApp({ directories, solutions }) {
           <Route exact path="/feature_directory">
             <FeatureDirectory addBasePath={addBasePath} directories={directories} />
           </Route>
-          <Route exact path="/">
+          <Route exact path="/legacy">
```

Comment:
+1. i think the overlay service can be used too

Reply:
I think since we're keeping current home as default and putting the next home behind a config toggle, this is unnecessary at this point. Maybe when we make it the default experience, this will be a concern.